### PR TITLE
markdown changes for dev center docs

### PR DESCRIPTION
**Pull request by KarateCowboy**

### CODE REVIEW

File: commands/run.js [hunk 1]
```diff
@@ -31,15 +31,14 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'run',
   description: 'run a one-off process inside a heroku dyno',
-  help: `
-Examples:
+  help: `#### Examples:
```
Comment (dickeyxxx) [inline commands/run.js]: we shouldn't add this kind of encoding unless we have a way to strip it back out I feel

**Comment by KarateCowboy** [comment]
Why, pray tell?

**Comment by dickeyxxx** [comment]
Because markdown encoding characters should not be displayed in CLI help

**Comment by KarateCowboy** [comment]
Well, I figured that was the reasoning, but why not, pray tell?

**Comment by dickeyxxx** [comment]
just because it's ugly. You should be able to add these characters in the script though by looking for "Examples?:"

**Comment by KarateCowboy** [comment]
@dickeyxxx bump for review

File: commands/inside.js [hunk 2]
```diff
@@ -26,8 +26,7 @@ module.exports = {
   command: 'inside',
   hidden: true,
   description: 'run a one-off process inside an existing heroku dyno',
-  help: `
-Examples:
+  help: `Examples:
```
Comment (dickeyxxx) [inline commands/inside.js]: you tool should trim the string like the CLI does

**Pull request merged**